metadetect/lsst/skysub.py

```diff
@@ -3,13 +3,124 @@
     SubtractBackgroundTask,
     SubtractBackgroundConfig,
     SourceDetectionTask,
-    SourceDetectionConfig,
 )
+from lsst.meas.algorithms import SourceDetectionConfig as OriginalSourceDetectionConfig
+from lsst.pex.config import Config, ConfigurableField, Field
+from lsst.pex.exceptions import RuntimeError as LSSTRuntimeError
+from lsst.pipe.base import Task, TaskError
 
 from .defaults import DEFAULT_THRESH
 from . import util
 
 
+class SourceDetectionConfig(OriginalSourceDetectionConfig):
+    @property
+    def thresholdValue(self):
+        return self.thresholdValue
+
+    @thresholdValue.setter
+    def thresholdValue(self, value):
+        self.thresholdValue = value
+
+
+SourceDetectionTask.ConfigClass = SourceDetectionConfig
```

Comment:
This bit of config boilerplate is repeated from the measure module. If one instance of this code needs to be changed, does the other one need to be changed too? If yes, then we should have this bit of boilerplate appear only once.